packages/compass-e2e-tests/helpers/selectors.ts

```diff
@@ -714,6 +714,11 @@ export const AggregationRestultsPrevPageButton =
 export const AggregationResultsCancelButton =
   '[data-testid="pipeline-results-loader-button"]';
 export const AggregationEmptyResults = '[data-testid="pipeline-empty-results"]';
+export const AggregationWriteOperationConfirmationModal =
+  '[data-testid="write-operation-confirmation-modal"]';
+export const AggregationWriteOperationConfirmButton = `${AggregationWriteOperationConfirmationModal} button:first-of-type`;
+export const AggregationWriteOperationCancelButton = `${AggregationWriteOperationConfirmationModal} button:last-of-type`;
```

Comment:
There are testids on those elements that we can use instead:

```suggestion
export const AggregationWriteOperationConfirmButton = `${AggregationWriteOperationConfirmationModal} button:[data-testid*="confirm-button"]`;
export const AggregationWriteOperationCancelButton = `${AggregationWriteOperationConfirmationModal} [data-testid*="cancel-button"]`;
```